### PR TITLE
feat: yamlfmt-golint-eau

### PR DESCRIPTION
**Pull request by edotau**
# Description

- Small PR to remove a few annoying golintci settings like import order, etc.
- A few clean ups removing whitespace at end of functions that dont add a whole lot of value
- I've reduce the lint rules to these 3, please feel free to adjust them as it fits the labs goals/standards in future PRs
    - https://github.com/vertgenlab/gonomics/blob/ee5acb189c76b676d41b59dc7467ba66ad23e77f/.golangci.yml#L17-L20
- Added yamlfmt which is used the same way on yaml files

## Relevant Links

- google's yamlfmt which is the same functionality as go fmt but on yaml files contained inside the repo
- https://github.com/google/yamlfmt
- lintci configuration documentation can be found here:
    - https://golangci-lint.run/usage/linters/
### Checklist before requesting a review

- [x] I performed a self-review of my code
- [x] If it this a core feature, I have added thorough tests
- [x] The command `go fmt` was used on all files included

### Testing

None

### Screenshots & Media

None

### Edge cases / Breaking Changes / Known Issues

None

### CODE REVIEW

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -61,6 +58,6 @@ jobs:
         uses: codecov/codecov-action@v3
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
-          fail_ci_if_error: false
+          fail_ci_if_error: true
```
Comment (edotau) [inline .github/workflows/go.yml:61]: Should this be true or false? @dasnellings

Comment (dasnellings) [reply .github/workflows/go.yml:61]: I believe we decided this should be false. We want to be notified if coverage is going down, but we don't want to outright fail the PR.

**Review by cfauci — APPROVED** [review]

**Review by RavenYL — APPROVED** [review]
